### PR TITLE
:wrench: Fix CHANGELOG

### PR DESCRIPTION
**Pull request by Sabnock01**
Fixes incorrect release versions in `CHANGELOG.md`.

### CODE REVIEW

**Comment by wilsoncusack** [comment]
were these actually all off by 1? 🤔 we have some PRs to automate this flow and improve

**Comment by Sabnock01** [comment]
> were these actually all off by 1? 🤔 we have some PRs to automate this flow and improve

`2` was missing and `5` was present twice.

**Review by roninjin10 — APPROVED** [review]

**Comment by wilsoncusack** [comment]
ok, 6 is also missing but doesn't have to be solve here

**Review by wilsoncusack — APPROVED** [review]

**Review by wilsoncusack — CHANGES_REQUESTED** [review]
actually, given 6 is missing, we should sanity check that all of these actually have the changes they say they do. E.g is writeFinalize... in 5 ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,19 +6,19 @@
 
 - Export writeFinalizeWithdrawTransaction and fix getProveWithdrawalTransactionArgs
 
-## 0.0.1-alpha.5
```
Comment (wilsoncusack) [inline CHANGELOG.md:9]: @Sabnock01 I think read finalizedWithdrawals first shows up here https://www.npmjs.com/package/op-viem/v/0.0.1-alpha.5?activeTab=code

I think possibly .2 was a no-op and the latest should have been .6, matching https://www.npmjs.com/package/op-viem/v/0.0.1-alpha.5?activeTab=versions

Comment (Sabnock01) [reply CHANGELOG.md:9]: Got it. So the only issue was the most recent was incorrectly labeled `.5` instead of `.6`.

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**